app/assets/datatables/jquery.dataTables.css

```diff
@@ -0,0 +1,452 @@
+/*
```

Comment:
It looks like you both include the file with Sprockets *and* through a CDN *and* via wrapper gem. Only one of these is necessary, but definitely don't copy/paste this big snippet of code. Just pick one and stick with it.